### ISSUE TITLE
Logo for Interactive

### ISSUE THREAD

**Issue by WhiteBlackGoose** [asc-community/AngouriMath#545]
**The package I want to suggest the idea to**: AngouriMath.Interactive

The current [logo](https://github.com/asc-community/AngouriMath/blob/master/Sources/Wrappers/AngouriMath.Interactive/icon.png) includes Jupyter logo which is incorrect (since you can use that pkg in VSC Notebooks or in console etc.).

[Here](https://drive.google.com/file/d/1Y8TwuEjL0-YO2DCE6c_qs8TrhGJES_tL/view?usp=sharing) are Paint.NET sources for the current logo. It should be changed to reflect that it's interactive (maybe .NET Interactive's [repo](https://github.com/dotnet/interactive) will help?).

**Comment by gulyapulya** [comment]
Hello @WhiteBlackGoose, this is my Hacktoberfest but I would want to contribute. I understand the issue with the current logo but I still have some questions about what the result should be. Do you want me to get rid of the Jupyter logo from the current logo and add some visuals to refer to the .NET interactive?

**Comment by WhiteBlackGoose** [comment]
@gulyapulya yeah, precisely! I myself can't really think of something good-looking :sweat_smile: 
[Here](https://github.com/asc-community/AngouriMath/blob/master/Sources/icon.png) is the general AM logo. I think it would be good to add something .NET Interactive related on top of it, for AngouriMath.Interactive

**Comment by Rafael-SOWNet** [comment]
**Triage, 2026-08-11.** Recommended for closure, and labelled `Opinions wanted` rather than closed because declining a proposal is a maintainer's call.

A logo for AngouriMath.Interactive is not engineering, and the package already ships an icon.

This is one of thirteen dormant proposals swept in the same pass. If you disagree with any of them the label is the place to say so; if you agree, closing is the action. Nothing here is a defect report, so nothing is lost by closing except the record of the idea, which this comment preserves.

**Comment by Happypig375** [comment]
This is already done so closing
